interfaces/apparmor/backend.go

```diff
@@ -511,20 +511,13 @@ func addContent(securityTag string, snapInfo *snap.Info, opts interfaces.Confine
 }
 
 func reloadProfiles(profiles []string, profileDir, cacheDir string) error {
```

Comment:
I wonder if this should move to `apparmor.go` now - given that unloadProfiles and loadProfiles are located there as well - WDYT @zyga ?

Reply:
Yes, that feels right (move to `apparmor.go`)